UKHomeOffice/ide-controller
# COP 3279: Image comparison table threshold

## Description
This branch updates the image comparison table tags with the new score thresholds, configurable in the score config file. 

## To test
- pull branch and run the command `npm ci && npm run build`
- deploy build to dev device
- scan a series of documents that return various "No Data", "Pass" and "Failure" results
- tag should display with appropriate colour, found on the IDE prototype. 

## Developer Checklist

\* Required
